xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -636,6 +644,7 @@ CVideoPlayer::CVideoPlayer(IPlayerCallback& callback)
   m_HasVideo = false;
   m_HasAudio = false;
   m_UpdateStreamDetails = false;
+  m_messageQueueTimeSize = VP_MESSAGE_QUEUE_TIME_SIZE;
```

Comment:
I find the "timesize" a bit confusing despite not being much familiar with the vp timing internals (probably refers to a scale applied to the time size?). Wouldn't it make more sense to use std::chrono for the constexpr and then use the absolute values in seconds/milliseconds/wtv depending on the actual use?

Reply:
> Wouldn't it make more sense to use std::chrono for the constexpr and then use the absolute values in seconds/milliseconds/wtv depending on the actual use?

I'm not sure if worth it: ms seems only used in two places vs 7 or 8 places that wants seconds (double).  While std::chrono is easy to convert to int with `.count()` i'm not sure if to convert to double requires additional `static_cast` anyway.  

Now only rounded values ​​8.0 or 4.0 are used but with the low latency PR they will probably want to try lower values ​​like 0.5s which would be cumbersome (for chrono) if everything is not implemented from the beginning in ms... I see it is more practical for now in seconds and double.